src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/mcfooter/MixedContentFooter.java

```diff
@@ -76,4 +109,48 @@ public int countArticleCards () {
     return articleLinks.size();
   }
 
+  public int countArticlesInExploreCard() {
+    wait.forElementVisible(exploreWikisCard);
+    return exploreWikisList.size();
+  }
+
+  public void clickOnViewAllLinkInDiscussions() {
+    wait.forElementVisible(viewAllLink).click();
+  }
+
+  public boolean isDiscussions() {
+    return discussionsBody.isDisplayed();
+  }
+
+  public String getUsername() {
+    return avatarUsername.getText();
+  }
+
+  public MixedContentFooter clickUserAvatar() {
+    avatarImage.click();
+    return this;
+  }
+
+  public MixedContentFooter clickDiscussionsPost() {
+    discussionsPost.click();
+    return this;
+  }
+
+  public boolean isZeroState() {
+    return discussionsZeroState.isDisplayed();
```

Comment:
`wait.forElementVisible()` should be used before

Reply:
done